addons/base_import_module/tests/test_cloc.py

```diff
@@ -0,0 +1,28 @@
+# -*- coding: utf-8 -*-
+# Part of Odoo. See LICENSE file for full copyright and licensing details.
+
+from odoo.tools import cloc
+from odoo.addons.base.tests import test_cloc
+
+class TestClocFields(test_cloc.TestClocCommon):
+
+    def test_fields_from_import_module(self):
+        """
+            Check that custom computed fields installed with an imported module
+            is counted as customization
+        """
+        # Make sure previous test is still working with base_import_module installed
+        self.common_test_field_count()
+        self.env['ir.module.module'].create({
+            'name': 'imported_module',
+            'state': 'installed',
+            'imported': True,
+        })
+        f1 = self.create_field('x_imported_field')
+        self.create_xml_id('import_field', f1.id, 'imported_module')
+        cl = cloc.Cloc()
+        cl.count_customization(self.env)
+        self.assertEqual(cl.code.get('imported_module', 0), 1, 'Count fields with xml_id of imported module')
+        f2 = self.create_field('x_base_field')
+        self.create_xml_id('base_field', f2.id, 'base')
```

Comment:
Shouldn't it be done *before* the call to `count_customization`?

Reply:
No but I forgot to recount with a new cloc object. thx for the catch